src/cell.cpp

```diff
@@ -23,19 +29,61 @@ Cell::Cell() {
 
 	rand.instantiate();
 
+	// Add CellStructures
+
+	// Load a CellStructure scene
+	Ref<PackedScene> nucleus_scene = ResourceLoader::get_singleton()->load("res://nucleus.tscn");
+	// Instantiate the scene and cast it to the specific type
+	Nucleus *nucleus = Object::cast_to<Nucleus>(nucleus_scene->instantiate());
+	// Add the CellStructure pointer to _cellStructures and as a child under this Cell
+	_cellStructures.push_back(nucleus);
+	this->add_child(nucleus);
+
+	Ref<PackedScene> mitochondria_scene = ResourceLoader::get_singleton()->load("res://mitochondria.tscn");
+	Mitochondria *mitochondria = Object::cast_to<Mitochondria>(mitochondria_scene->instantiate());
+	_cellStructures.push_back(mitochondria);
+	this->add_child(mitochondria);
+
+	Ref<PackedScene> ribosomes_scene = ResourceLoader::get_singleton()->load("res://ribosomes.tscn");
+	Ribosomes *ribosomes = Object::cast_to<Ribosomes>(ribosomes_scene->instantiate());
+	_cellStructures.push_back(ribosomes);
+	this->add_child(ribosomes);
+
+	Ref<PackedScene> flagella_scene = ResourceLoader::get_singleton()->load("res://flagella.tscn");
+	Flagella *flagella = Object::cast_to<Flagella>(flagella_scene->instantiate());
+	_cellStructures.push_back(flagella);
+	this->add_child(flagella);
+
 	_spriteSize = Size2();
 }
 Cell::~Cell() {}
```

Comment:
Since child nodes are being added with `add_child`, they should be freed with `queue_free()` when the cell is deconstructed.